src/io/rename_at.rs

```diff
@@ -20,8 +18,8 @@ impl Op<RenameAt> {
     pub(crate) fn rename_at(from: &Path, to: &Path, flags: u32) -> io::Result<Op<RenameAt>> {
         use io_uring::{opcode, types};
 
-        let from = driver::util::cstr(from)?;
-        let to = driver::util::cstr(to)?;
+        let from = super::util::cstr(from)?;
```

Comment:
Maybe six of one, half a dozen of the other. I could see using *crate* instead of *super* here since we'll want to access the util modules from all over, whether or not they are just one crate up and over or not.

Reply:
Oh man. My comment is wrong. This util isn't at the top level, it's under io/, so *super* is totally correct.